integration-tests/integration_tests/integration_tests/amq/mhs_inbound_queue.py

```diff
@@ -0,0 +1,8 @@
+import os
+
+from comms.blocking_queue_adaptor import BlockingQueueAdaptor
+
+MHS_INBOUND_QUEUE = BlockingQueueAdaptor(os.environ.get('MHS_SECRET_INBOUND_QUEUE_USERNAME', None),
+                                         os.environ.get('MHS_SECRET_INBOUND_QUEUE_PASSWORD', None),
```

Comment:
Is that default value of None necessary here? Couldn't we just use i.e. get('MHS_SECRET_INBOUND_QUEUE_USERNAME')

Reply:
The default is none since this value is not required to run tests against the rabbitmq container. This is current/working functionality for MHS so we should leave it as-is